api/src/main/java/eu/coatrack/api/ServiceApi.java

```diff
@@ -205,6 +205,14 @@ public void setEntryPoints(List<EntryPoint> entryPoints) {
         this.entryPoints = entryPoints;
     }
 
+    /**
+     *
+     * @return a simple String to display the Service API in a GUI
+     */
+    public String getGuiStringRepresentation() {
+        return String.format("%s / %s (%s)", name, uriIdentifier, localUrl);
```

Comment:
Bugfix works fine, however, to be consistent with other screens, we should re-format this string the same way that we used in other places, i.e. `Service Name (service-uri-identifier -> https://url)`

There are two options, feel free to choose what works better: 

- adapt this Java method to format the string according to the usual pattern
- or remove this method for good and instead modify the html templates and use Thymeleaf expressions to build the display string

Reply:
I think that as you already have the method done and working, the first option would be faster to apply

Reply:
> the first option would be faster to apply

I agree, however I assume that the second option would be more unified with the rest of the code.

Reply:
```getGuiStringRepresentation()``` was removed and Thymeleaf expressions were used instead as suggested.